PlexRequests.UI/Modules/SearchModule.cs

```diff
@@ -179,11 +179,20 @@ private Response RequestMovie(int movieId)
             Log.Trace(movieInfo.DumpJson);
             //#if !DEBUG
 
+            var settings = PrService.GetSettings();
+
+            // check if the movie has already been requested
             Log.Info("Requesting movie with id {0}", movieId);
-            if (RequestService.CheckRequest(movieId))
+            var existingRequest = RequestService.CheckRequest(movieId);
+            if (existingRequest != null)

```

Comment:
I'm doing that check right below. I needed the logic to be this way to return the proper message.